### ISSUE TITLE
examples, tutorials

### ISSUE THREAD

**Issue by JernejL** [BeRo1985/pasjson#1]
This looks like a great library, it's used by your gltf work and in your other projects, but in the end there is no proper documentation. 
 
I would like to make a few examples for others to use and use the library in my projects but i have no idea where to approach using it, and what method of usage to suggest - so if you have any kind of test case for the code that you used, i can start with that - otherwise i'll do my best to figure it out from PasGLTF.

**Comment by coderextreme** [comment]
Agreed!  I'm trying simple things like this in the Castle Engine fork, and it's not working!

```function MyJSON.JSONParse(): TDOMDocument;
  var
    //aStream: TStream;
    JSONItem: TPasJSONItem;
  begin
    //aStream := TStringStream.Create('{}');
    JSONItem:=TPasJSON.Parse('{}',[],TPasJSONEncoding.AutomaticDetection);
    TPasJSON.Stringify(JSONItem);
    Result := nil;
  end;```
